### PR TITLE
Made myft-digest prefs setting more exciting

### PR DESCRIPTION
**Pull request by laurieboyes**
new

Core remains intact

### CODE REVIEW

File: client/main.scss [hunk 1]
```diff
@@ -21,7 +21,7 @@ $output-critical: true !default;
 @import "n-section/main";
 
 @include oColorsSetUseCase(myft-page-cold, all, 'cold-1');
-@include oColorsSetUseCase(myft-newsletter, all, 'teal-1');
+@include oColorsSetColor('myft-newsletter-preference-background', #F4EADA);
```
Comment (laurieboyes) [inline client/main.scss]: Pretty sure this is criminal but put it in for the sake of discussion. In the design, this colour was an origami colour with an opacity. This was fine until I realised we had to overlay this other panel with the same colour - doh!

It definitely doesn't seem worth having this colour in origami, or even in n-ui, as it'll only be used for myFT. Does anyone have any thoughts?

Comment (keirog) [inline client/main.scss]: Although not perfect, you could do something like this: https://github.com/Financial-Times/n-card/pull/47/files

Comment (wheresrhys) [inline client/main.scss]: Can't we change the design? Saying yes to using new colours for a single use case is always a slippery slope

Comment (keirog) [inline client/main.scss]: It’s technically not a new colour though, if I‘m understanding correctly… It’s more to do with opacity issues

Comment (laurieboyes) [inline client/main.scss]: Yep, we're back on teal-1

**Comment by keirog** [comment]
Might be worth checking with mattH/designux channel (if you haven’t already) that the animation fits with site-wide conventions. Same for the close button I guess.

Otherwise 👍 ✨

**Pull request merged**